### DISCUSSION TITLE
true WYSIWYG for Joomla's tinyMCE Editor

### DISCUSSION THREAD

**Discussion by jpg13150** [templaza/astroid-framework#330] [Q&A]
Hello
I tried using this plugin with tinymce
I modified the files as described in the documentation but it doesn't work
Can you help me
Thank you

https://wysiwyg.dgrammatiko.dev/

**Comment by darekmer** [comment]
Actually, it's a nice thing. In Cassiopeia it works, in Zero, One doesn't
![tinymce](https://github.com/templaza/astroid-framework/assets/44267306/939d6800-dd0a-4082-8ca9-6d702d88965f)

**Comment by Chacapamac** [comment]
I personally use JCE, completely WYSIWYG.
